tests/acceptance/test_organization_discover.py

```diff
@@ -82,7 +82,6 @@ def test_save_query_edit(self):
             self.browser.wait_until_not(".loading")
             self.browser.find_element_by_xpath("//button//span[contains(text(), 'Save')]").click()
             self.browser.get(self.path + "saved/1/?editing=true")
-            self.browser.wait_until_test_id("result")
```

Comment:
This test is actually already, and has been failing, but something about my change has caused it to _actually_ fail now. I tried pretty hard to investigate this. For some reason before my changes it would double load.

I'm not convinced this is right and we may have to revisit.

Reply:
nevermind this was (somewhat) legitimate. I did accidentally change some error handling logic.